code/game/objects/items/hand_items.dm

```diff
@@ -0,0 +1,605 @@
+		var/atom/throw_target = get_edge_target_turf(sucker, owner.dir)
+		sucker.throw_at(throw_target, 6, 3, owner)
+	else
+		owner.visible_message(span_danger("[owner] bops [sucker] with [owner.p_their()] [src.name]!"), span_danger("You bop [sucker] with your [src.name]!"), \
+			span_hear("You hear a dull thud!"), ignored_mobs=list(sucker))
+		sucker.apply_damage(15, STAMINA)
+		log_combat(owner, sucker, "bopped", src.name, "(setup)")
+		to_chat(sucker, span_userdanger("[owner] bops you with [owner.p_their()] [src.name]!"))
+	qdel(src)
+
+
+/obj/item/hand_item/noogie
+	name = "noogie"
+	desc = "Get someone in an aggressive grab then use this on them to ruin their day."
+	icon_state = "latexballon"
+	inhand_icon_state = "nothing"
+
+/obj/item/hand_item/noogie/attack(mob/living/carbon/target, mob/living/carbon/human/user)
+	if(!istype(target))
+		to_chat(user, span_warning("You don't think you can give this a noogie!"))
+		return
+
+	if(HAS_TRAIT(user, TRAIT_PACIFISM))
+		to_chat(user, span_warning("You can't bring yourself to noogie [target]! You don't want to risk harming anyone..."))
+		return
+
+	if(!(target?.get_bodypart(BODY_ZONE_HEAD)) || user.pulling != target || user.grab_state < GRAB_AGGRESSIVE || user.getStaminaLoss() > 80)
+		return FALSE
+
+	var/obj/item/bodypart/head/the_head = target.get_bodypart(BODY_ZONE_HEAD)
+	if((target.get_biological_state() != BIO_FLESH_BONE && target.get_biological_state() != BIO_JUST_FLESH) || !the_head.is_organic_limb())
+		to_chat(user, span_warning("You can't noogie [target], [target.p_they()] [target.p_have()] no skin on [target.p_their()] head!"))
+		return
+
+	// [user] gives [target] a [prefix_desc] noogie[affix_desc]!
+	var/brutal_noogie = FALSE // was it an extra hard noogie?
+
+	var/prefix_desc = "rough"
+	var/affix_desc = ""
+	var/affix_desc_target = ""
+
+	if(HAS_TRAIT(target, TRAIT_ANTENNAE))
+		prefix_desc = "violent"
+		affix_desc = "on [target.p_their()] sensitive antennae"
+		affix_desc_target = "on your highly sensitive antennae"
+		brutal_noogie = TRUE
+
+	if(user.dna?.check_mutation(/datum/mutation/human/hulk))
+		prefix_desc = "sickeningly brutal"
+		brutal_noogie = TRUE
+
+
+	var/message_others = "[prefix_desc] noogie[affix_desc]"
+	var/message_target = "[prefix_desc] noogie[affix_desc_target]"
+
+	user.visible_message(span_danger("[user] begins giving [target] a [message_others]!"), span_warning("You start giving [target] a [message_others]!"), vision_distance=COMBAT_MESSAGE_RANGE, ignored_mobs=target)
+	to_chat(target, span_userdanger("[user] starts giving you a [message_target]!"))
+
+	if(!do_after(user, 1.5 SECONDS, target))
+		to_chat(user, span_warning("You fail to give [target] a noogie!"))
+		to_chat(target, span_danger("[user] fails to give you a noogie!"))
+		return
+
+	if(brutal_noogie)
+		to_chat(user, span_warning("You are not supposed to see this!")) // LOBOTOMYCORPORATION ADDITION - i needed some code here so it wouldn't give warnings, enjoy dis
```

Comment:
brutal_noogie variable is local, just... don't add it?

Reply:
for future reviewers: Keeping the brutal_noogie variable for less possible future conflicts with cherry-picking from TG. Conversation on discord